changelog.yaml

```diff
@@ -52,6 +52,16 @@
   date: TBD
   changes:
 
+    - type: enhancement
+      impact: minor
+      title: Make K8s API connection timeout configurable
+      description: |-
+        When stewardci-core hits a network issue when connecting to API server, it will only recover after 15 min when the default timeout occurs.
+        To improve the situation the timeout should be configurable. Following parameters are added now for run- and tenant-controllers:
```

Comment:
Be more precise here. 
Distinguish parameter for command line and configuration in helm chart in values.yaml.

Reply:
Fixed here https://github.com/SAP/stewardci-core/pull/305/commits/ac1bcdec92944c5f061d9a25468f0763ca462385